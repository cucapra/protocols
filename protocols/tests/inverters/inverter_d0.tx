// ARGS: --verilog inverters/inverter_d0.v --protocol inverters/inverter_d0.prot
// RETURN: 101
invert(0, 1);
