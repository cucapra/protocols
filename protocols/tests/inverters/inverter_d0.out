error: Thread 0 attempted conflicting assignment to 'a': current=ValueOwned(11111111111111111111111111111111), new=ValueOwned(00000000000000000000000000000000)
  ┌─ inverters/inverter_d0.prot:7:3
  │
7 │   DUT.a := DUT.s;
  │   ^^^^^^^^^^^^^^^ Thread 0 attempted conflicting assignment to 'a': current=ValueOwned(11111111111111111111111111111111), new=ValueOwned(00000000000000000000000000000000)

